app/driver/settings_travis.py

```diff
@@ -0,0 +1,147 @@
+
+"""
+Django settings for DRIVER project testing.
```

Comment:
As discussed, may be worth trying to add a try/except when loading the secrets file with a dict of defaults when it's not available. Then the travis settings file can just import everything and override only what it needs.